tools/Az/Az.psd1

```diff
@@ -64,18 +64,18 @@ RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.6.4'; },
                @{ModuleName = 'Az.Billing'; RequiredVersion = '1.0.1'; }, 
```

Comment:
missing modules for Az.DataBoxEdge and Az.SqlVirtualMachine

Reply:
Added. Thanks